quick-start.md

```diff
@@ -216,7 +216,15 @@ data:
 ...
 ```
 
-Deploy: `kubectl apply -f deployments/quick-start/kubernetes-vault.yaml`
+- For kubernetes versions >= 1.6 deploy: 
+
+  ```kubectl apply -f deployments/quick-start/kubernetes-vault-rbac.yaml```
+
+  Special care is required here when you are deploying to a different namespace. Please modify the file directly
```

Comment:
Maybe update this to: Special care is required here when you are deploying to a namespace other than `default`.